export const reminderList = () => {
	try {
		return "reminder";
	} catch (e) {
		console.error("Error:", e);
	}
};
